bootstrap/bootstrap.go

```diff
@@ -1279,16 +1279,29 @@ func (b *Bootstrap) defaultCheckoutPhase() error {
 		addRepositoryHostToSSHKnownHosts(b.shell, b.Repository)
 	}
 
+	var err error
 	var mirrorDir string
 
 	// If we can, get a mirror of the git repository to use for reference later
 	if experiments.IsEnabled(`git-mirrors`) && b.Config.GitMirrorsPath != "" && b.Config.Repository != "" {
 		b.shell.Commentf("Using git-mirrors experiment 🧪")
-		var err error
-		mirrorDir, err = b.updateGitMirror()
-		if err != nil {
-			return err
+
+		// Skip updating the Git mirror before using it?
+		if b.Config.GitMirrorsSkipUpdate {
+			mirrorDir = filepath.Join(b.Config.GitMirrorsPath, dirForRepository(b.Repository))
```

Comment:
it'd be nice here to add another comment saying that we're skipping the mirror update

Reply:
Sure, comment added in c36f7946ecbbc8976f3fb13e8707956c717376bf